src/HotChocolate/Core/src/Types/Types/InputField.cs

```diff
@@ -23,11 +23,11 @@ public InputField(InputFieldDefinition definition, FieldCoordinate fieldCoordina
             DefaultValue = definition.DefaultValue;
             Property = definition.Property;
 
-            if (definition.Formatters.Count == 0)
+            if (definition.GetFormatters().Count == 0)
             {
                 Formatter = null;
             }
-            else if (definition.Formatters.Count == 1)
+            else if (definition.GetFormatters().Count == 1)
```

Comment:
var formatters = definition.GetFormatters();

if(formatters.Count == 0)
and so on....